infrastructure/terraform/dns/main.tf

```diff
@@ -0,0 +1,32 @@
+locals {
+  weco_hosted_zone_id = "Z3THRVQ5VDYDMC"
+}
```

Comment:
You could also access this with an `aws_route53_zone` data source: https://www.terraform.io/docs/providers/aws/d/route53_zone.html

That’s what we do in the Archivematica stack: https://github.com/wellcomecollection/archivematica-infra/blob/33563750240d667fb6e0ea349919be8dc08b44ae/terraform/infra/certs.tf#L19-L23

Reply:
Nice - thank you!